Dockerfile

```diff
@@ -19,4 +21,4 @@ ADD . /src
 WORKDIR /src
 
 # Install Hokusai
-RUN pip install .
+RUN pipenv install --deploy
```

Comment:
This seems to be wrong actually. So `pip install .` runs the `setup.py` script which seems to build the project. What is the root purpose of this... to make the `hokusai` command available?

Reply:
I used the below as recommended in `pipenv`'s docs to install the setup script locally. This makes it available as a command when you install....

```
Install a local setup.py into your virtual environment/Pipfile:
   $ pipenv install -e .
```

You know, I'm not sure if this is a good thing or not, ha. Hopefully it'd make this part work fine though.